archive-press_quote.php

```diff
@@ -0,0 +1,71 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+if( have_posts() ) {
+  $i = 0;
+
+  while( have_posts() ) {
+    the_post();
+    $quote = get_post_meta($post->ID, '_igv_quote_text', true);
+    $link = get_post_meta($post->ID, '_igv_quote_link', true);
+
+    $title = get_the_title();
```

Comment:
I was unsure about this in the metaboxes PR but then figured that there are no permalinks and titles don't have to be unique. Only issue would be in the admin working out which was which. We could solve that by adding the quote to the admin list view for quotes, or we could not worry about it, or they could add the month or issue number or something else in the title